README.md

```diff
@@ -121,7 +121,7 @@ Install the `openshift-client` and `s2i-java` tasks from the catalog repository
 
 ```bash
 $ oc create -f https://raw.githubusercontent.com/openshift/tektoncd-catalog/release-v0.7/openshift-client/openshift-client-task.yaml
-$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-catalog/release-v0.7/s2i-java-8/s2i-java-8-task.yaml
+$ oc create -f https://raw.githubusercontent.com/openshift/tektoncd-catalog/release-next/buildah/buildah.yaml
 
```

Comment:
These tasks are installed by default. The tutorial should explain that a number of tasks are installed on OpenShift by default and point out the URL of those tasks for review but no need to create the actual tasks.